DNN Platform/DotNetNuke.Web.Mvc/DnnMvcDependencyResolver.cs

```diff
@@ -41,14 +29,11 @@ public DnnMvcDependencyResolver(IServiceProvider serviceProvider, IDependencyRes
         /// </returns>
         public object GetService(Type serviceType)
         {
-            try
-            {
-                return _serviceProvider.GetService(serviceType);
-            }
-            catch
-            {
-                return _resolver.GetService(serviceType);
-            }
+            var scope = HttpContext.Current?.GetScope();
+            if (scope != null)
+                return scope.ServiceProvider.GetService(serviceType);
```

Comment:
This looks right to me, I will need review some of the docs on scoped services before I could give any type of approval for adding scoped resolutions.

The only question that jumps out at me is if the scope is null instead of throwing an exception maybe we can have fallback logic. I think if we are all in on scoped containers that is unnecessary. (more to come later)

Reply:
I'm with @ahoefling on this one, I think this is ok, but looking to confirm

Reply:
see https://github.com/dnnsoftware/Dnn.Platform/pull/3520#issuecomment-578186734 for my attempt to explain the need of a request scope.